src/ensembl/src/content/app/App.tsx

```diff
@@ -61,6 +67,14 @@ const App = (props: AppProps) => {
           <Route path={`/`} component={HomePage} exact />
           <Route path={`/global-search`} component={GlobalSearch} />
           <Route path={`/species-selector`} component={SpeciesSelector} />
+          <Route
+            path={`/entity-viewer-interstitial`}
+            component={EntityViewerInterstitial}
+          />
+          <Route
+            path={`/genome-browser-interstitial`}
+            component={BrowserInterstitial}
+          />
```

Comment:
No, that's not right. The interstitials should live in the GenomeBrowser and EntityViewer apps. Same as how we are now showing the example links when the user visits these apps for the first time. They should not have their own urls.